attributes/network/cddl/11_ledger_tokens.cddl

```diff
@@ -106,19 +143,18 @@ tokens.update@param = {
     ; Servers MAY error if this field is updated to invalid values.
     ? 3 => transaction-fee,
 
-    ; A memo associated with the token. This will override the memo, not just
-    ; add to it.
-    ? 4 => memo,
-
     ; An owner for the token.
     ; To remove the owner of a token, this can be null.
-    ? 5 => address / null,
-
-    ; An optional visual logo. See 2_ledger.cddl for more information on logos.
-    ? 6 => visual-token-logo,
+    ? 4 => address / null,
 
     ; A memo associated with this update.
-    ? 7 => memo,
+    ? 5 => memo,
+
+    ; Extended info to add. This is merged with the existing extended info.
+    ? 6 => token-extended-info,
+
+    ; Extended info to remove.
+    ? 7 => [ + attribute-related-index ],
```

Comment:
Out of curiosity, why did you decide to use two fields?

Reply:
You might want to add new info, or delete some infos, without touching the rest (since it could be a lot of data). How would you see this otherwise? A record containing two fields?

Reply:
As discussed, I would try to be consistent with the rest of the spec, either by re-passing the whole list, like we do for every other field we want to update, or by adding new endpoints as we do with features.